ArduCopter/autoyaw.cpp

```diff
@@ -5,12 +5,7 @@ Mode::AutoYaw Mode::auto_yaw;
 // roi_yaw - returns heading towards location held in roi
 float Mode::AutoYaw::roi_yaw()
 {
-    roi_yaw_counter++;
-    if (roi_yaw_counter >= 4) {
-        roi_yaw_counter = 0;
-        _roi_yaw = get_bearing_cd(copter.inertial_nav.get_position(), roi);
-    }
-
+    _roi_yaw = get_bearing_cd(copter.inertial_nav.get_position(), roi);
```

Comment:
Why are we still storing this?  Can we eliminate the member variable?

Reply:
yep the _roi_yaw could be remove

Reply:
done